charts/stateful/templates/scaled-object.yaml

```diff
@@ -0,0 +1,12 @@
+{{- $name := include "argonaut-deployment.uname" . -}}
+{{- $chartName := .Chart.Name -}}
+{{- $releaseName := .Release.Name -}}
+{{- $releaseService := .Release.Service -}}
+{{- $env := .Values.argonaut.env -}}
+{{- $version := .Values.version | default .Release.Revision -}}
+
+{{- if .Values.scaledObjectAuth -}}
+{{ toYaml .Values.scaledObjectAuth }}
+---
```

Comment:
line separator should be after {{ end }} block unless it is required to be logically present there

Reply:
Can we have a line separator above the `scaledObject` block even if there no other block above it?

Wasn't sure of the syntax. I'll make this small change.

Reply:
just to be clear - what you wrote is perfectly valid. It is a style choice.